AdminUi/apps/admin_ui/lib/core/widgets/filters/date_filter.dart

```diff
@@ -31,10 +32,9 @@ class _DateFilterState extends State<DateFilter> {
           children: [
             DropdownButton<FilterOperator>(
               value: _operator,
-              onChanged: (newValue) {
-                if (newValue!.userFriendlyOperator.isNotEmpty) {
-                  widget.onFilterSelected(_operator = newValue, _selectedDate);
-                }
+              onChanged: (selectedOperator) {
+                if (selectedOperator!.userFriendlyOperator.isEmpty) return;
```

Comment:
why do you check for a userFriendlyOperator?

why do you use `!` without checking if it is defined?

```suggestion
                if (selectedOperator == null) return;
```

this would be the right way, I think I commented like that.